src/GameplaySystem.cpp

```diff
@@ -11,11 +11,50 @@ GameplaySystem::GameplaySystem() {
 
 void GameplaySystem::update() {
     // Update game state here
+    EventSystem::queue_event (
+        Event (
+            EventType::IDLE_GAME_OBJECT_EVENT,
+            "object_id", 1,
+            "rotation_rad", 0.01f
+        )
+    );
 }
 
 void GameplaySystem::handle_load(const Event& e) {
     // Initialize gameplay here
     std::cout << "Gameplay initialized here" << std::endl;
+
+    // Ship 1
+    EventSystem::queue_event(
+        Event(
+            EventType::ADD_GAME_OBJECT_EVENT,
+            "object_id", 0,
```

Comment:
I think if we have to manually set these there's gonna be alot of mistakes made with game objects having conflicting values. 

Let's create a singleton that just spits out the next gameobject id. So if any one ever needs to generate a gameobject id, they can just use that and we'll never worry about conflicting gameobject ids.